obspy/signal/spectral_estimation.py

```diff
@@ -53,6 +54,88 @@
 NOISE_MODEL_FILE = os.path.join(os.path.dirname(__file__),
                                 "data", "noise_models.npz")
 
+earthquakes = {
+    (1.5,10): [[7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 
+                1.1300000e+01, 2.2600000e+01],
+               [1.1940722e-06, 5.9652811e-06, 4.2022150e-05, 2.1857010e-04, 
+                6.4011669e-04, 1.0630361e-03]],
+    (1.5,100): [[1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 1.1300000e+01, 
+                 2.2600000e+01],
+               [2.0636508e-07, 1.0591860e-06, 4.4487964e-06, 1.1678490e-05, 
+                1.2427714e-05]],
```

Comment:
@FMassin check out the CircleCI log (thats the code formatting check). Many trailing whitespaces in this part and most of the continuation lines with square brackets need to be indented one space to match the above.
All the dict keys miss a space after comma.

Reply:
I just did, fixed.